include/drjit-core/jit.h

```diff
@@ -495,13 +495,17 @@ extern JIT_EXPORT uint32_t jit_registry_id(const void *ptr);
 
 /// Return the largest instance ID for the given domain
 /// If the \c domain is \c nullptr, it returns the number of active entries in
-/// all domains for the given variant.
+/// all domains for the given variant. If the \c variant is empty as well, it
```

Comment:
```suggestion
/// all domains for the given variant. If the \c variant is \c nullptr as well, it
```

Need to be precise since `""` and `nullptr` are not the same thing.